src/parameterized_tendencies/radiation/radiation.jl

```diff
@@ -467,14 +467,14 @@ function radiation_tendency!(Yₜ, Y, p, t, radiation_mode::RadiationDYCOMS)
     q_tot_isoline = FT(0.008)
     Operators.column_reduce!(
         (nt1, nt2) ->
-            abs(nt1.q_tot - q_tot_isoline) < abs(nt2.q_tot - q_tot_isoline) ?
-            nt1 : nt2,
-        isoline_z_ρ_q,
-        Base.broadcasted(NT ∘ tuple, ᶜz, Y.c.ρ, ᶜspecific.q_tot),
+            abs(specific(nt1.ρq_tot, nt1.ρ) - q_tot_isoline) <
```

Comment:
As above. It seems strange to access \rho q_tot from 'NT' and from 'Y.c' below. I'd prefer to avoid what looks to me like a duplication. Same for \rho itself.